### ISSUE TITLE
Add SSL to backend API

### ISSUE THREAD

**Issue by kylepjohnson** [cltk/cltk_api#10]
I've been seeing a lot about [Let's Encrypt](https://letsencrypt.org/) from the EFF. I think this would be a great way to go.

An easy tutorial I'll probably follow: https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-14-04

**Comment by manu-chroma** [comment]
@kylepjohnson I would like to implement this feature. Can you tell me where to start ?

**Comment by kylepjohnson** [comment]
@manu-chroma Have you worked with Flask before? We need it to run with Gunicorn + Nginx, then do these steps.

**Comment by manu-chroma** [comment]
@kylepjohnson Yes, I do have experience with flask beforehand. Don't have much experience with gunicorn though.

**Comment by kylepjohnson** [comment]
@manu-chroma If you want this ticket, I will spin up a new server for you with instructions about what we need done. Let me know and I'll assign this to you.

**Comment by manu-chroma** [comment]
@kylepjohnson I think I can work on this. Please assign and tell me what needs to be done.

**Comment by kylepjohnson** [comment]
I'll let you know by email once I put up a dev server for you. This will take me a day or two.

**Comment by modassir** [comment]
@manu-chroma ask me if you are stuck anywhere, as I have already done the same thing with gunicorn, nginx and Let's Encrypt for my dev server.

Reference: [Gunicorn and nginx](https://www.digitalocean.com/community/tutorials/how-to-serve-flask-applications-with-gunicorn-and-nginx-on-ubuntu-14-04)
and the one @kylepjohnson [referred](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-14-04)

**Comment by kylepjohnson** [comment]
Thank you @modassir, we'll move faster working together. Appreciate it. KJ
